### ISSUE TITLE
Create layer architecture

### ISSUE THREAD

**Issue by Felizolinha** [deeplearningunb/Sign-Language-Digits-Detector#16]
## Description

Plan out the layer architecture and implement it on the ipynb.

**Comment by Felizolinha** [comment]
I think we can start with a basic convolutional network, with 3 convolutional layers (each layer with a Conv2D and MaxPooling2D). At the end we flatten it, and get the outputs by feeding it into a Dense layer with many neurons and then provide it to the last layer, which will contain 10 neurons, one for each digit.

For the loss function we can use categorical crossentropy, by considering each digit as a category.

**Comment by luis-gustavo** [comment]
We also need to add [Batch Normalization](https://towardsdatascience.com/batch-normalization-in-neural-networks-1ac91516821c/) and [Droupout Layers](https://keras.io/layers/core/) between the convolutional layers, so we can have a better generalization and to reduce overfitting.
